ansible-ipi-install/roles/installer/tasks/main.yml

```diff
@@ -0,0 +1,11 @@
+---
```

Comment:
While I'd like to see lots more tags, think adding them to these top level task would help in controlling the flow when needed

Reply:
agreed.

Reply:
#132 Feature Enhancement